src/coreclr/src/gc/gcpriv.h

```diff
@@ -4875,9 +4873,9 @@ BOOL heap_segment_unmappable_p (heap_segment* inst)
 }
 
 inline
-BOOL heap_segment_loh_p (heap_segment * inst)
+BOOL heap_segment_uoh_p (heap_segment * inst)
 {
-    return !!(inst->flags & heap_segment_flags_loh);
+    return !!(inst->flags & (heap_segment_flags_loh));
```

Comment:
>heap_segment_flags_loh [](start = 29, length = 22)

method name is changed but not the flag name?

Reply:
The name is correct. It is just that we have only one kind of UOH at the moment. 
It will be `return !!(inst->flags & (heap_segment_flags_loh | heap_segment_flags_poh));`   when POH is added.

Yes, parens are not needed until we have more flags.

---
In reply to: [350582752](https://github.com/dotnet/runtime/pull/235#discussion_r350582752) [](ancestors = 350582752)